ADOPTERS.md

```diff
@@ -1,5 +1,6 @@
 # Linkerd 2.x adopters
 
+- [Advance Latam] (http://www.advlatam.com/)
```

Comment:
There is no space needed here. Also can you sign your commits by following https://github.com/linkerd/linkerd2/pull/5097/checks?check_run_id=1276151820

Reply:
@Pothulapati changes implemented